modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc

```diff
@@ -77,21 +92,23 @@ The `sep` and `eol` will be used to convert the `DataFrame` to delimited data an
 |Lizzy| 19|female|
 +-----+---+------+
 
+.Example: Post Request to TigerGraph
+[source, gsql]
+http://host:port/restpp/ddl/Social?tag=load_Person&filename=file1
+--data <delimited_data>
```

Comment:
This is a bit of a mystery / not well presented.  I think the author was trying to show how to run the loading job.
My complaints:
1.  There is no sentence that explains what is it. ("But I wrote a heading...")
2. The heading is vague. Some of our developers say "Post" as a synonym for "Loading Data using a specially designed endpoint, which uses POST mode." But no one outside their clique will understand that. 
3. It doesn't match our running example. It has "load_Person" as the job name, but our jon is called "load_Social".
4. I don't understand why `--data <delimited>` appears, because our data is coming from `filename=file1`

In general, you can run a loading job using GSQL (`RUN LOADING JOB ...`) or as a REST endpoint, as shown.

Reply:
<img width="841" alt="Screen Shot 2023-11-03 at 2 53 48 PM" src="https://github.com/tigergraph/server-docs/assets/144745015/bf3da0d7-58a0-4867-b0d8-5da12c4ec7b2">

For now decided to comment out the post request and instead lead the user to our other Documentation on loading jobs. This way they can equip themselves to build their own.